org.eclipse.xtext.tests/xtend-gen/org/eclipse/xtext/xtext/Issue1161Test.java

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) 2019 itemis AG (http://www.itemis.eu) and others.
```

Comment:
The generated Java file should reflect the change in the Xtend file, too. Was the Xtend file compiled?

Reply:
Done